dashboard/public/data/users/Simba Khadder.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "Simba Khadder",
+  "type": "User",
+  "default-version": "default-version",
+  "all-versions": ["default-version"],
+  "versions": {
+    "default-version": {
+      "metadata": {
+        "version-name": "first-version",
+        "created": "2020-08-09T13:49:51.141Z",
+        "visibility": "private",
+        "teams": ["Sample model team", "data orchestration"],
+        "permissions": "Admin"
+      },
+      "data": {
+        "training sets": [
+          {
+            "name": "Wine Quality Dataset",
+            "description": "set of wine quality samples and quality ratings",
+            "tags": ["wine", "batch", "quality"],
+            "variants": "default"
+          }
+        ],
+        "features": [
+          {
+            "name": "Non-free Sulfur Dioxide",
+            "description": "Total non-free sulfur in sample",
+            "tags": ["wine", "batch", "sulfur"],
+            "variants": "default"
+          },
+          {
+            "name": "fixed_acidity",
+            "description": "fixed acidity of smaple",
+            "tags": ["wine", "batch", "sulfur"],
+            "variants": "default"
+          },
+          {
+            "name": "clean_density",
+            "description": "Clean density of sample",
+            "tags": ["wine", "batch", "sulfur"],
+            "variants": "default"
+          },
+          {
+            "name": "LogAvgPurchasePrice",
+            "description": "Average purchase price of sample",
+            "tags": ["purchase", "commerce", "payment"],
+            "variants": "default"
+          }
+        ],
+        "labels": [
+          {
+            "name": "Wine quality rating",
+            "description": "Quality rating of wine sample",
+            "tags": ["wine", "batch", "sulfur"],
+            "variants": "default"
+          },
+          {
+            "name": "Wine spoiled",
+            "description": "Whether or not wine is spoiled",
+            "tags": ["wine", "batch", "sulfur"],
+            "variants": "default"
+          }
+        ],
+        "models": [
+          {
+            "name": "wine random forest",
+            "description": "Random forest wine classification model",
+            "tags": ["wine", "batch", "sulfur"],
+            "variants": "default"
```

Comment:
Shouldn't variants be a list in this case?

Reply:
Yes. Currently there's no strict implementation of variant select on the object page. Since this change will apply for more than just model I will handle it in another PR.

Reply:
Added basic variant select to new commit